src/client/components/MealDetail.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { MealContext } from "./MealContext";
+import Navbar from "./Navbar";
+import Footer from "./Footer";
+import ShowReviews from "./ShowReviews";
+import ReservationForm from "./ReservationForm";
+import PostReviewForm from "./PostReviewForm";
+const mealsUrl = process.env.REACT_APP_MEALS_URL;
+
+function MealDetail() {
+  const { id } = useParams();
+  const { meals } = useContext(MealContext);
+  const meal = meals.find((meal) => meal.id === Number(id));
```

Comment:
Such a component that is meant to display details of a single meal - if I understood correctly - should definitely have a single meal passed into it as props.
Why would it need to know about all the meals and filter them? 🤔